### DISCUSSION TITLE
make queue settings of grouped players accessible

### DISCUSSION THREAD

**Discussion by DutchJaFO** [music-assistant/support#836] [Feature requests and ideas]
Possible fix for : https://github.com/music-assistant/hass-music-assistant/issues/815

MA automatically groups my Sonos Speakers if I group them in my Sonos App as well.
This is a very cool feature.

However since all these speakers still have their own queues and related settings it is impossible to know 
(a) what settings are actually in use

(b) check and change these 

Part of the problem appears to be that the speaker that is listed as 'in control' (or at least the one that is showing the name) is semi-randomly chosen.

I discovered that while I had set one speaker queue to  mp3 / small chunks (in advanced settings) when I looked at the queue today the setting was FLAC / big chunks ...
I also have the current weird situation that the MA shows an empty queue, when in reality there is a MA based queue playing.
HA => Music Assistant  : 
![image](https://user-images.githubusercontent.com/39138434/184434680-4ae012ae-a0dd-4632-8c9a-fff08bcb71f2.png)

Sonos app confirming that I've got music playing :
![image](https://user-images.githubusercontent.com/39138434/184434864-eacb77dd-0046-4717-b486-f4c1f5a029bb.png)

I believe that this may be the cause for the issue that I had posted as a bug.

**Comment by OzGav** [comment]
Relates to v1
